ogiste/ireporter
# Ft ui create incident 163186991

#### What does this PR do?
Adds UI integration with API endpoint to allow user to create an incident using the UI

#### Description of Task to be completed?
A user should be able to create an inciden using the UI

#### How should this be manually tested?
Navigate to [create an incident page](http://ogiste.github.io/ireporter/UI/create_quest.html) after signing in on the [homepage](http://ogiste.github.io/ireporter/UI/index.html) and provide the new incident details to create a new incident a pop up message will verify the results of the creation with any error or success message.

#### What are the relevant pivotal tracker stories?
[#163186991](https://www.pivotaltracker.com/story/show/163186991)
